src/winter_onboarding_2021/4clojure/alisha.clj

```diff
@@ -22,4 +22,6 @@
 
 (defn problem14 [] 8)
 
-(defn problem15 [] (fn [x] (* x 2)))
\ No newline at end of file
+(defn problem15 [] (fn [x] (* x 2)))
+
+(defn problem16 [] (fn [x] (str "Hello, " x "!")))
```

Comment:
Same, why is it returning a function here?

Reply:
So @govindkrjoshi instead of defining like this I could have used #(str "Hello, " % "!") but then this is also an anonymous function right? Am I missing something?